Overview.src.html

```diff
@@ -4113,8 +4114,9 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   </ol>
 </ol>
 
-<p>The <dfn title=dom-Headers><code>Headers(<var>init</var>)</code></dfn> constructor,
-when invoked, must run these steps:
+<p>The
+<dfn title=dom-Headers data-export data-dfn-for=Headers data-dfn-type=constructor data-lt="Headers(init)"><code>Headers(<var>init</var>)</code></dfn>
```

Comment:
Also, constructors and methods generally shouldn't be referenced elsewhere. Maybe in examples and such...